.github/workflows/main.yml

```diff
@@ -43,12 +43,12 @@ jobs:
         with:
           node_version: 16
 
+      - name: Build packages
+        run: pnpm build
+
       - name: Run lint
         run: pnpm lint
 
-      - name: Run format
```

Comment:
I believe this code was added by @yukukotani  yesterday. Could you confirm if it's appropriate to delete it?

Reply:
Sorry, I don't get why this should be removed.

> Note that I think most of projects are relying on ESLint to make sure all the files are prettiered but this repo does not seem to integrate Prettier with ESLint in that way, here I am assuming that is intentional

IMO, as @naruaway said, our ESLint configuration doesn't integrate with Prettier, so we have to manually check the format using `pnpm format`.

Reply:
In addition, integrating Prettier with ESLint is generally not recommended.

ref: https://prettier.io/docs/en/integrating-with-linters#notes

Reply:
Based on what @yukukotani said, I'm inclined to think that this code should remain as it is. If there's any particular reason for removing it, could you please share it with me? @naruaway

Reply:
Thanks for the info, this was my mistake. I initially thought ESLint is covering Prettier so I removed this line but then I realized our ESLint is not configured in that way as I mentioned in PR description. And then I forgot to push the change to restore this "Run format" 😭 

I now restored it and rebased 👍 

> integrating Prettier with ESLint is generally not recommended.

Oh good to know! And the rationale explained in the Prettier doc totally makes sense